Adding doc on asset objects (#21)
podium-lib/podium-lib.github.io: docs/api/document.md

## Customizing

> Podium ships with a default document template which should cover most uses. It

Reviewer: use cases

> Podium ships with a default document template which should cover most uses. It

Reviewer 2: ```suggestion
Podium ships with a default document template which should cover most use cases. It
```

## Request Properties

> A document template will need properties which are request  bound. This can be

Reviewer 2: ```suggestion
A document template will need properties which are request bound. This can be
```

<!--END_DOCUSAURUS_CODE_TABS-->

## Assets

On the [`HttpIncoming`](incoming.md) object which is passed on to the document
template one can find an array of `AssetCSS` objects on the `.css` property and
an array of `AssetJS` objects on the `.js` property . These properties hold the

> assets of a podlet or a layout. In a layout they can additionally to the layouts

Reviewer 2: ```suggestion
assets of a podlet or a layout. In a layout they can, in addition to the layouts
```

> assets, hold the assets of the requested podlets.

Reviewer: In a layout they can hold the assets of the requested podlets in addition to the assets of the layout itself.
